src/Sinks/FluentdSink.cs

```diff
@@ -160,73 +130,52 @@ private void Format(LogEvent logEvent, StringWriter output)
                     name = '@' + name;
                 }
 
-                output.Write(',');
-                JsonValueFormatter.WriteQuotedJsonString(name, output);
-                output.Write(':');
-                new JsonValueFormatter("$type").Format(property.Value, output);
+                ((IDictionary<string, object>)localEvent).Add(name, "");
+                var visitor = new SerilogVisitor();
+                visitor.Visit((IDictionary<string, object>)localEvent, name, property.Value);
             }
 
-            output.Write('}');
+            var serializationContext = new SerializationContext(PackerCompatibilityOptions.PackBinaryAsRaw) { SerializationMethod = SerializationMethod.Map };
```

Comment:
Wonder if we should create one instance of the serialization context

Reply:
Yes